CMakeLists.txt

```diff
@@ -13,27 +13,27 @@ include_directories("lambdatwist-p3p/utils")
 
 find_package(Ceres REQUIRED )
 
-add_library(libpnp utils/mlibtime.cpp pnp_ransac.cpp p4p.cpp)
-target_include_directories(libpnp PRIVATE ${CERES_INCLUDE_DIRS})
-target_link_libraries(libpnp ${CERES_LIBRARIES})
-#target_compile_features(libpnp PUBLIC cxx_std_11)
+add_library(pnp utils/mlibtime.cpp pnp_ransac.cpp p4p.cpp)
+target_include_directories(pnp PRIVATE ${CERES_INCLUDE_DIRS})
+target_link_libraries(pnp ${CERES_LIBRARIES})
+#target_compile_features(pnp PUBLIC cxx_std_11)
 
 add_executable(main test_pnp.cpp)
-target_link_libraries(main libpnp)
+target_link_libraries(main pnp)
 
-install(TARGETS libpnp
+install(TARGETS pnp
     LIBRARY DESTINATION lib
     ARCHIVE DESTINATION lib)
 
-install(DIRECTORY include/libpnp
+install(DIRECTORY .
```

Comment:
Need to do this a better way. Just did this for now to install all the files to the staging directory.